api/tests/identity-access-management/integration/infrastructure/repositories/user.repository.test.js

```diff
@@ -1728,6 +1728,23 @@ describe('Integration | Identity Access Management | Infrastructure | Repository
         // then
         expect(error).to.be.instanceOf(UserNotFoundError);
       });
+
+      it('update username with empty username', async function () {
```

Comment:
Suggestion pour mieux documenter ce que fait le test : 

```suggestion
      context('when username is null', function () {
        it('still updates the user with the null username', async function () {
```

Également, pour avoir une logique thématique dans les tests, déplacer ce test au-dessus du précédent, de manière à ce qu'il soit juste sous le test `updates the username`.